src/backend/joanie/settings.py

```diff
@@ -590,6 +596,11 @@ class Test(Base):
         "django.contrib.auth.hashers.MD5PasswordHasher",
     ]
     USE_SWAGGER = True
+    DEVELOPER_EMAIL = values.Value(
+        "developer@example.com",
+        environ_name="DEVELOPER_EMAIL",
+        environ_prefix=None,
+    )
```

Comment:
From my point we should hardcode this value in Test configuration in order to prevent some test to fail if they do some assertions related to this setting.